src/App.tsx

```diff
@@ -149,65 +154,67 @@ const Providers: React.FC = ({ children }) => {
-                                              <DpiIndexComponentsProvider>
-                                                <MviTokenMarketDataProvider>
-                                                  <MviComponentsProvider>
-                                                    <BedTokenMarketDataProvider>
-                                                      <BedIndexComponentsProvider>
-                                                        <DataTokenMarketDataProvider>
-                                                          <DataComponentsProvider>
-                                                            <IndexTokenMarketDataProvider>
-                                                              <V3FarmingProvider>
-                                                                <StreamingFeeProvider>
-                                                                  <TokenSupplyProvider>
-                                                                    {children}
-                                                                  </TokenSupplyProvider>
-                                                                </StreamingFeeProvider>
-                                                              </V3FarmingProvider>
-                                                            </IndexTokenMarketDataProvider>
-                                                          </DataComponentsProvider>
-                                                        </DataTokenMarketDataProvider>
-                                                      </BedIndexComponentsProvider>
-                                                    </BedTokenMarketDataProvider>
-                                                  </MviComponentsProvider>
-                                                </MviTokenMarketDataProvider>
-                                              </DpiIndexComponentsProvider>
-                                            </DpiTokenMarketDataProvider>
-                                          </Btc2xFliTokenSupplyCapProvider>
-                                        </Btc2xFliPortfolioDataProvider>
-                                      </Btc2xFliTokenMarketDataProvider>
-                                    </Eth2xFliTokenSupplyCapProvider>
-                                  </Eth2xFliIndexPortfolioDataProvider>
-                                </Eth2xFliTokenMarketDataProvider>
-                              </BuySellProvider>
-                            </PricesProvider>
-                          </MviStakingRewardsProvider>
-                        </FarmingTwoProvider>
-                      </FarmingProvider>
-                    </BalancesProvider>
-                  </ExternalAirdropProvider>
-                </RewardsProvider>
-              </AirdropProvider>
-            </MediaQueryProvider>
-          </ApolloProvider>
+          <DAppProvider config={config}>
```

Comment:
The only line change here is adding `<DAppProvider config={config}>`. Every line below just gets indented by one tab

Reply:
soon we will kill some of these providers